components/BottomNavigation/tests/unit/BottomNavigationBarColorThemerTests.m

```diff
@@ -18,6 +18,7 @@
 #import "MDCBottomNavigationBarColorThemer.h"
 #import "MaterialColorScheme.h"
 #import "MaterialBottomNavigation.h"
+#import "../../src/private/MDCBottomNavigationItemView.h"
```

Comment:
I'd be surprised if this builds with bazel. You may have to add a "private" target for your unit tests.

Example:

```

mdc_objc_library(
    name = "private",
    hdrs = native.glob(["src/private/*.h"]),
    includes = ["src/private"],
    visibility = ["//visibility:private"],
)

swift_library(
    name = "unit_test_swift_sources",
    srcs = glob(["tests/unit/*.swift"]),
    deps = [
        ":Snackbar",
        ":private",
        ":ColorThemer",
        ":TypographyThemer",
    ],
    visibility = ["//visibility:private"],
)
```

Reply:
There already is a private target our tests depend on, we're already importing this header within BottomNavigationTests.m